### PR TITLE
Mi439: 20240709

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Mi439.json [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-                        "paypal": "",
-                        "telegram": "",
-                        "dt": "https://github.com/crdroidandroid/android_device_xiaomi_Mi439",
-                        "common-dt": "https://github.com/crdroidandroid/android_device_xiaomi_mithorium-common",
-                        "kernel": "https://github.com/crdroidandroid/android_kernel_xiaomi_msm8937"
-                }
-        ]
+	"response": [
+		{
+			"maintainer": "",
```
Comment (gwolf2u) [inline Mi439.json]: missing maintainer tag

**Review by gwolf2u — CHANGES_REQUESTED** [review]
also, why so many indentations?

File: Mi439.json (hunk 1 above)
Comment (ghost) [reply Mi439.json]: Uhh, seems like i somehow pushed the wrong file after editing the config🤦

**Comment by ghost** [comment]
> also, why so many indentations?

Probably a issue with previous file, i simply copied and pushed this one so it must be okay.

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**